### PR TITLE
add FlowToken upgrade plan

### PR DESCRIPTION
**Pull request by joshuahannan**
Adds a plan for upgrading the FlowToken contract to use the destructor conditional
Still needs the code argument

### CODE REVIEW

File: transactions/update-contract/2022/aug-3/arguments.json [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"type":"String","value":"FlowToken"},{"type":"String","value":""}]
```
Comment (joshuahannan) [inline transactions/update-contract/2022/aug-3/arguments.json:1]: @pgebheim Do you know what to use to generate the second argument here from the smart contract code?

File: transactions/update-contract/2022/aug-3/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Update Service Account Contract
+
+> August 3, 2022
+
+## Contract to be deployed
+
+https://github.com/onflow/flow-core-contracts/blob/master/contracts/FlowToken.cdc
+
+## Dapper Labs Builds
```
Comment (joshuahannan) [inline transactions/update-contract/2022/aug-3/README.md:9]: I copied from a previous upgrade here and just changed some names and dates. I'm not sure who should be the ones doing the building and signing or if we use a tool now

File: transactions/update-contract/2022/aug-3/arguments.json (hunk 1 above)
Comment (pgebheim) [reply transactions/update-contract/2022/aug-3/arguments.json:1]: I don't!

I'm asking @janezpodhostnik  :-D

Comment (janezpodhostnik) [reply transactions/update-contract/2022/aug-3/arguments.json:1]: The second argument should be the hex encoded contract. Its going to get used here: https://github.com/onflow/service-account/blob/main/templates/update_contract.cdc

**Comment by joshuahannan** [comment]
Closing this because it seems like Kan is already doing the upgrade

**Pull request closed without merging**